sites/docs/components/PropListing.svelte

```diff
@@ -18,22 +60,47 @@
 			}, 2000);
 		} catch (err) {}
 	}
+
+	let counter = 0;
+	const updateIdName = () => {
+		idName = name.replace(/[^a-zA-Z0-9]/g, '-');
+		while (names.has(idName)) {
+			counter++;
+			idName = counter > 1 ? `${name}-${counter}` : `${name}`;
+		}
```

Comment:
Is this going to be stable when multiple of the same component appear?

This looks good to me, but is probably a good candidate for unit testing though